package/lean/v2ray/Makefile

```diff
@@ -280,7 +291,7 @@ ifneq ($(CONFIG_V2RAY_EXCLUDE_V2CTL),y)
 	mv -f $(GO_PKG_BUILD_BIN_DIR)/main $(GO_PKG_BUILD_BIN_DIR)/v2ctl
 
 ifeq ($(CONFIG_V2RAY_COMPRESS_UPX),y)
-	$(STAGING_DIR_HOST)/bin/upx --lzma --best $(GO_PKG_BUILD_BIN_DIR)/v2ctl
+	$(STAGING_DIR_HOST)/bin/upx --lzma --best $(GO_PKG_BUILD_BIN_DIR)/v2ctl || true
```

Comment:
Same.

Reply:
Ignore the error and continue to compile

Reply:
> Ignore the error and continue to compile

So is there any example? I don't ever see any error with existing command.

Reply:
For example, upx is missing or compression error, this is only to prevent upx error

Reply:
All right, please bump PKG_RELEASE.

Reply:
updated